src/nl/rubensten/texifyidea/util/PsiUtil.kt

```diff
@@ -531,10 +531,23 @@ fun LatexEndCommand.beginCommand(): LatexBeginCommand? = previousSiblingOfType(L
  */
 fun LatexContent.isDisplayMath() = firstChildOfType(LatexDisplayMath::class) != null && firstChildOfType(LatexEnvironment::class) == null
 
+/*
+ * Technically it's impossible to determine for all cases whether a users wants to compile with biber or biblatex.
+ * But often when people use the biblatex package they use biber.
+ * And often, when they use biber they use \printbibliography instead of \bibliography.
+ * Hence, the following methods often work - and if they don't, users can easily change the compiler in the run config.
+ */
 
 /**
  * Checks if the fileset for this file has a bibliography included.
  *
  * @return `true` when the fileset has a bibliography included, `false` otherwise.
  */
 fun PsiFile.hasBibliography() = LatexIncludesIndex.getItemsInFileSet(this).any { it.name == "\\bibliography" }
+
+/**
+ * Checks if the fileset for this file uses \printbibliography, in which case the user probably wants to use biber.
+ *
+ * @return `true` when the fileset has a bibliography included, `false` otherwise.
+ */
+fun PsiFile.usesBiber() = LatexIncludesIndex.getItemsInFileSet(this).any { it.name == "\\printbibliography" }
```

Comment:
Please change (both occurences of) `LatexIncludesIndex.getItemsInFileSet(this)` to the `commandsInFileSet()` extension.

I want to push to use these extension functions in favour of the static variants. It looks cleaner, is better understandable and hides implementation.

Reply:
Agree, I just didn't know this extension function existed. I added a note in the kdoc, as I guess you don't want to deprecate `getItemsInFileSet` (yet)?

Reply:
Definitely not. It's the underlying function with the actual behaviour.